packages/hebao_v1/contracts/base/BaseModule.sol

```diff
@@ -75,6 +75,11 @@ contract BaseModule is ReentrancyGuard, Module
         _;
     }
 
+    receive() external payable
+    {
+        revert("UNSUPPORTED");
```

Comment:
I think this is pretty much the same as not having the function.

Reply:
Ok, removed.